streams/src/test/java/org/apache/kafka/test/MockProcessor.java

```diff
@@ -53,6 +53,11 @@ public void process(final K key, final V value) {
         delegate.process(new Record<>(key, value, context.timestamp(), context.headers()));
     }
 
+    // TODO check needs updating
```

Comment:
Why not just call `delegate.process(record)`? Actually, why do we need to add this method here at all?